bin/main.rb

```diff
@@ -103,40 +113,66 @@ def display_board(cells)
 name1 = ui.prompt_player_name('player 1')
 symbol1 = 'X'
 array1 = []
-player1 = Player.new(name1, symbol1, array1)
+player1 = Player.new(name1.magenta, symbol1.magenta, array1)
 name2 = ui.prompt_player_name('player 2')
 symbol2 = 'O'
 array2 = []
-player2 = Player.new(name2, symbol2, array2)
-current_player = 'player1'
-
-board = Board.new(current_player)
-game_logic = GameLogic.new(player1, player2, board.cell)
-ui.display_board board.cell
+player2 = Player.new(name2.cyan, symbol2.cyan, array2)
+again="Y"
 
-until game_logic.game_end
+  while again == "Y"
+  current_player = 'player1'
```

Comment:
The current_player can be initialized with the game initialization method

Reply:
I dont even use a current player anymore

Reply:
see my new comment